src/js/main.js

```diff
@@ -172,6 +205,58 @@ var SupportKit = Marionette.Object.extend({
         return this.throttledUpdate();
     },
 
+    track: function(eventName) {
+        this._checkReady();
+
+        var hasEvent = this._hasEvent(eventName),
+            rulesContainEvent = this._rulesContainEvent(eventName);
+
+        if (rulesContainEvent) {
+            this._eventCollection.create({
+                name: eventName,
+                user: this.user
+            }, {
+                success: _.bind(this._checkConversationState, this)
+            });
+        } else if (!rulesContainEvent && !hasEvent) {
+            this._createEvent(eventName);
```

Comment:
That'd be harsh on the unit tests and make that async. I could create `_ensureEventExists` and call `_hasEvent` in there.